core/SCsub

```diff
@@ -120,6 +120,8 @@ if env['builtin_zstd']:
         "compress/zstd_ldm.c",
         "compress/zstd_opt.c",
         "compress/zstdmt_compress.c",
+		"compress/zstd_compress_literals.c",
```

Comment:
Indentation issue (mix of tabs and spaces).

Reply:
I think this is GitHub UI bug as I saw this too, but when I actually when into the file to view it, the indentation was normal.

Reply:
No, indentation is about characters, not visuals. The indentation here should be 8 spaces, but you used 4 spaces + 1 tab. If you configure your tab display to something different from 4 characters width, you'll see the problem.

Reply:
Oh I see, the problem was with my editor then. Fixed it now, sorry about that.